### ISSUE TITLE
Training “Network in Network model” described on  Model-Zoo 

### ISSUE THREAD

**Issue by dabilied** [BVLC/caffe#1475]
As described at "https://github.com/BVLC/caffe/wiki/Model-Zoo"  Network in Network model is very
small and easy to be trained.So,I trained it on imagenet data.However,it seems doesn't wrok. 
ENVIRONMENT:Tesla K20Xm; NVIDIA-SMI 340.32; cuda 6.5; cudnn-6.5-linux-R1;
when I was training this net ,the  ilsvrc12_val_lmdb is also used for training another net ,can this influence? 
Have some one encountered this type of problem and has solution?

Iteration 100000, Testing net (#0)
     Test net output #0: accuracy = 0.000999999
     Test net output #1: loss = 6.92628 (\* 1 = 6.92628 loss)
My prototxt:
solver:(this is almost same as model-zoo described)
net: "ninImagenetTrainVal.prototxt"
test_iter: 5000
test_interval: 5000
base_lr: 0.01
lr_policy: "step"
gamma: 0.1
stepsize: 200000
display: 20
max_iter: 450000
momentum: 0.9
weight_decay: 0.0005
snapshot: 50000
snapshot_prefix: "models/ninImagenet"
solver_mode: GPU
device_id: 0
ninImagenetTrainVal.prototxt:
In this file,I only set the crop_size: 227 ,test batch_size 50 ,and rename some layers names(such as name: "relu0" -> name: "relu1", this makeing the relu layer names continuous in net ,"relu4" was disappear on the web description).

**Comment by ProGamerGov** [comment]
@dabilied As someone who has never trained a model before but has some experience with caffe, can you explain how I take my data set of images like [these](http://people.bath.ac.uk/hc551/dataset.html) and then use them to train a NIN model?
